OpenSim/Actuators/ClutchedPathSpring.cpp

```diff
@@ -122,6 +122,19 @@ void ClutchedPathSpring::setInitialStretch(double stretch0)
      set_initial_stretch(getStretch(state));
  }
  
+ void ClutchedPathSpring::extendFinalizeFromProperties()
+ {
+     Super::extendFinalizeFromProperties();
+
+     OPENSIM_THROW_IF_FRMOBJ(get_stiffness() < 0,
```

Comment:
One more concern: will these exceptions really be thrown if the property values are `SimTK::NaN`? IIRC, `SimTK::NaN < 0` evaluates to `false`. Might need to do something like
```cpp
OPENSIM_THROW_IF_FRMOBJ(SimTK::isNaN(get_stiffness()) || get_stiffness() < 0,
                        ...);
```